### PR TITLE
[CI SKIP] Updated CHANGELOG and podspec for 1.5.1 release.

### PR DESCRIPTION
**Pull request by sid-github**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,30 @@
...
+### Compatibility
+* Requires iOS 8 as the minimum deployment target.
+* The EarlGrey gem runs out of the box for [Swift 3.0](https://docs.google.com/document/d/1AeleXccp35EUX4ILa6CT3CwlxLSZq1YLrco9JF27p9k/edit) and Swift 2.3.
+* Has been tested for support till iOS 10.01 on devices and simulator.
+
+### Bug Fixes
+* Fixed CI Ruby test for Carthage.
+
+### Enhancements
+* Improved touch injection for speed and to work independent of screen refresh rate.
```
Comment (tirodkar) [inline CHANGELOG.md]: This reads a little off:

Improved touch injection speed by making it work independent of the screen refresh rate.

Comment (sid-github) [inline CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,30 @@
...
+## [1.5.1](https://github.com/google/EarlGrey/tree/1.5.1) (11/04/2016)
+
+```
+Baseline: [d9eb1bc]
+   + [d9eb1bc]: Updated ChangeLog and pod spec for 1.5.1 release
+```
+
+### Compatibility
+* Requires iOS 8 as the minimum deployment target.
+* The EarlGrey gem runs out of the box for [Swift 3.0](https://docs.google.com/document/d/1AeleXccp35EUX4ILa6CT3CwlxLSZq1YLrco9JF27p9k/edit) and Swift 2.3.
```
Comment (khandpur) [inline CHANGELOG.md]: don't use google doc links. Use a shorter url which we can change later on?

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,30 @@
...
+* Has been tested for support till iOS 10.01 on devices and simulator.
+
+### Bug Fixes
+* Fixed CI Ruby test for Carthage.
+
+### Enhancements
+* Improved touch injection speed by making it work independent of the screen refresh rate.
+* Added synchronization for NSURLConnection::sendSynchronousRequest.
```
Comment (khandpur) [inline CHANGELOG.md]: nit: stylize apis using `NSURLConnection::sendSynchronousRequest`

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,30 @@
...
+* Added synchronization support for data URLs.
```
Comment (khandpur) [inline CHANGELOG.md]: we removed synchronization support for data urls...didn't add it. We skip any synchronization for urls that have the data scheme.

File: CHANGELOG.md (hunk 2 above)
Comment (tirodkar) [inline CHANGELOG.md]: Don't link this at all.

File: CHANGELOG.md (hunk 4 above)
Comment (sid-github) [inline CHANGELOG.md]: done

**Pull request merged**